### PR TITLE
Zee Spencer on Cohere and Zinc Collective

### PR DESCRIPTION
**Pull request by anaulin**

### CODE REVIEW

**Comment by anaulin** [comment]
@zspencer I'd love to have your feedback on the short intro I wrote for you, and the links to your personal presence. Here is how those parts of the episode will look like on the site:

Screen Shot 2021-01-07 at 7 09 17 PM

Do you have a headshot I could use to make the little graphic (the one with your name on it) a little less boring? 🙏 

I'm also happy for any feedback you have on the episode (including if you'd like anything removed, in which case I'm also happy to re-write Git history to leave no trace).

File: content/episode/zee-spencer-cohere-zinc-coop.md [hunk 1]
```diff
@@ -0,0 +1,614 @@
...
+
+{{< timestamp "[00:11:48]" >}}
+You can't sell your life like that. I mean, you can try, but at some point it just like breaks you. Or you get really rich and then you think it was all worth it. And you go on the speaking circuit and you like, you know, go on Hacker News, apparently, if that's still a thing. I don't know. I stopped reading the internet a while ago.
+
+{{< timestamp "[00:12:06]" >}}
+_Ana:_ Yeah, the orange site is still a thing. I do not recommend it. So you went back to that startup and you met some of your current business partners. So what happened?
+
+{{< timestamp "[00:12:17]" >}}
+_Zee:_ Yeah, so I went back to my old S-corp and I was like, "All right, I'm going to dust off the old Rolodex", an, an old colleague, Attursh, was like, "Hey, would you like to do some Rails work for Code for America?" And I was like, "that seems like a great way to stick the landing  and do something meaningful."
```
Comment (anaulin) [inline content/episode/zee-spencer-cohere-zinc-coop.md]: @zspencer Can you confirm what is the right spelling of "Attursh"'s name? (I have a strong feeling this has been badly mangled in this transcript...).

Comment (anaulin) [reply content/episode/zee-spencer-cohere-zinc-coop.md]: Fixed.

File: content/episode/zee-spencer-cohere-zinc-coop.md [hunk 2]
```diff
@@ -0,0 +1,619 @@
...
+
+{{< timestamp "[00:18:12]" >}}
+And I was like "THAT" like,  the East Bay Permanent Real Estate Cooperative was exactly what I was wanting to exist, but for technology companies. And so through the course of 2018, I was just looking at it like, okay. So what they do is they buy assets and they maintain those assets. And then they sell those assets to the people who live there.
+
+{{< timestamp "[00:18:30]" >}}
+Well, what if we did that with software products that no longer have a good home? So I bought an app from a friend of a friend and I'm like, Hey, this brings in about a thousand dollars a quarter, not a lot of money, but if we just buy it off you and then do a revenue share for a few years and we maintain it, we release updates. How does that sound? I'm like, yeah, that sounds great. Cause I'm not maintaining it. I'm not releasing updates.
```
Comment (zspencer) [inline content/episode/zee-spencer-cohere-zinc-coop.md]: This probably should be "He's like, yeah, that sounds great." I mis-spoke :).

File: content/episode/zee-spencer-cohere-zinc-coop.md [hunk 3]
```diff
@@ -0,0 +1,619 @@
...
+  * [Sustainable Economies Law Center](https://www.theselc.org/)
+  * [East Oakland Collective](https://www.eastoaklandcollective.com/)
+  * [Gregory Jackson](https://www.instagram.com/melaphonik/) and [Repaired Nations](http://repairednations.org/)
+  * [East Bay Permanent Real Estate Cooperative](https://ebprec.org/)
+
+Impact and cooperative business creation and investment:
+  * [The Next Egg](https://www.thenextegg.org/)
+  * [Jenny Kassan](https://www.jennykassan.com/) and [Angels of Main Street](https://www.angelsofmainstreet.com/)
+  * [Aniya Williams](https://twitter.com/operaqueenie) and [Zebras Unite](https://zebrasunite.coop/)
+  * [Kachuwa Impact Fund](https://www.kachuwaimpactfund.com/)
```
Comment (zspencer) [inline content/episode/zee-spencer-cohere-zinc-coop.md:28]: This list is missing Arlan's fund, Backstage Capital

File: content/episode/zee-spencer-cohere-zinc-coop.md (hunk 2 above)
Comment (anaulin) [reply content/episode/zee-spencer-cohere-zinc-coop.md]: Actually I think the transcript is just wrong, you are not saying "I'm" in that bit. Fixed.

File: content/episode/zee-spencer-cohere-zinc-coop.md (hunk 3 above)
Comment (anaulin) [reply content/episode/zee-spencer-cohere-zinc-coop.md:28]: Added!

**Pull request merged**